index.html

```diff
@@ -601,14 +601,14 @@ <h2>
                   <div class="col-lg-4 col-md-6 mentor-cards-home">
                     <div class="card">
                       <div class="img-bx">
-                        <img src="images/mentor/43.jpg" alt="img" />
+                        <img src="images/mentor/vidhi.jfif" alt="img" />
```

Comment:
Rename these files to number (whichever comes next in sequence) while resolving #2